### PR TITLE
Setup hello microverse

### PR DESCRIPTION
**Pull request by AbenezerKb**
In this PR, I :

- [ ] set up a "Hello world" repository
- [ ] Created a new branch called "my-second-branch", according to the GitHub flow
- [ ] Added a .gitignore file.
- [ ] Set up a linter for HTML and CSS
- [ ] Added a descriptive README file to your project
- [ ] Added 3 files (each one in a separate commit):
- [ ] 1. A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
- [ ] 2. A .css file that includes one class with styles for your header
- [ ] 3. A test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (zieeco) [inline .gitignore:1]: - KIndly add the `node_modules/` directory as you don't want it to be tracked by git also.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    - git installed on your machine
+    - code editor like vscode
+    - browser
+### Setup
+    - clone the project from this repo and open the index.html file
+### Install
+    - git
+
+
+## Authors
```
Comment (zieeco) [inline README.md:27]: - Please, kindly make the author singular as you've worked on this project alone to make your readme more professional.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @AbenezerKb,


Great job on setting up the Hello Microverse Project. 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Descriptive PR. ✔️
- Nice setting up the Project. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (AbenezerKb) [reply .gitignore:1]: Currently I don't have that folder since I have started the Javascript exercise. I don't think I should add it

**Review by ichala — APPROVED** [review]
Hi @AbenezerKb ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark: 
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966


Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### OPTIONAL CHANGES

**those are optional but highly recommended**

- [ ] even if you don't have the `node_modules/` folder, the requirements say it needs to be included in the `.gitignore` file, this is a reason to decline the pr and ask again for changes but since this is your last review and you did a GREAT effort on working in this I will approve it and please try to implement it in the future projects! great job and keep rocking! 👏👏👏👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**